integration_tests/src/main/python/date_time_test.py

```diff
@@ -474,6 +474,16 @@ def test_date_format(data_gen, date_format):
     assert_gpu_and_cpu_are_equal_collect(
         lambda spark : unary_op_df(spark, data_gen).selectExpr("date_format(a, '{}')".format(date_format)))
 
+@pytest.mark.parametrize('date_format', supported_date_formats, ids=idfn)
+# from 0001-02-01 to 9999-12-30
+@pytest.mark.parametrize('data_gen', [LongGen(min_val=int(datetime(1, 2, 1).timestamp()), max_val=int(datetime(9999, 12, 30).timestamp()))], ids=idfn)
```

Comment:
I tested the long value of  `0001-01-01` year and it reports 'year 0 is out of range'.
Do we need to update this case?

Reply:
LongGen does not filter the special cases to be within min_val and max_val.

https://github.com/NVIDIA/spark-rapids/blob/ae41666706201139068e48a7b2277e7436c4e530/integration_tests/src/main/python/data_gen.py#L289

So if 0, 1, or -1 were outside of min/max it would return a problem. But in this case I think we are okay because 0 is 1970 so 1 and -1 are fine.